src/pre_process/m_initial_condition.fpp

```diff
@@ -262,7 +266,7 @@ contains
 
                     ! STL patch
                 elseif (patch_icpp(i)%geometry == 21) then
-                    call s_model(i, patch_id_fp, q_prim_vf)
+                    call s_model(i, patch_id_fp, q_prim_vf, .false.)
```

Comment:
see above comment about putting a raw boolean in the middle of the argument list.